DEV-2/ReversedPolishNotation.java

```diff
@@ -0,0 +1,129 @@
+import java.util.LinkedList;
+
+/**
+ * Class counts value of input expression and print it
+ * Uses Reversed Polish Notation
+ * Can't count negative negative numbers and exp view of numbers
+ */
+public class ReversedPolishNotation {
+
+    private LinkedList<Double> stack = new LinkedList<>(); // List for numbers
+    private LinkedList<Character> operators = new LinkedList<>(); // List for operators
+    private String expression; //String for expression
+
+    /**
+     * Constructor for class
+     * Print counted value
+     * @param string
+     */
+    public ReversedPolishNotation(String string) {
+        if (string == "") {
+          System.out.println("Wrong expression");
+        }
+        this.expression = string;
+        System.out.println(parseAndCalculate());
+    }
+
+    /**
+     * @param c
+     * @return true if there is space-symbol
+     *         false if no space-symbol
+     */
+    private boolean isSpace(char c) {
+        return c == ' ';
+    }
+
+    /**
+     * @param c
+     * @return true if there is one of symbols: + - * /
+     */
+    private boolean isOperator(char c) {
+        return c == '+' || c == '-' || c == '*' || c == '/' || c == '%';
```

Comment:
not flexible and extendable way, every operation must be presented by separate class